src/hooks/useENSRecordDisplayProperties.tsx

```diff
@@ -56,28 +56,32 @@ export default function useENSRecordDisplayProperties({
     () => Object.keys(imageKeyMap).includes(recordKey),
     [recordKey]
   );
-  const isUrlValue = useMemo(
+
+  const isUrlRecord = useMemo(
     () =>
-      recordValue.match(/^http/) ||
       [ENS_RECORDS.url, ENS_RECORDS.website].includes(recordKey as ENS_RECORDS),
-    [recordKey, recordValue]
+    [recordKey]
   );
+  const isUrlValue = useMemo(() => recordValue.match(/^http/), [recordValue]);
 
   const url = useMemo(() => {
-    if (isUrlValue) {
+    if (isUrlValue || isUrlRecord) {
       return recordValue.match(/^http/)
         ? recordValue
         : `https://${recordValue}`;
     }
     if (links[recordKey]) {
       return `${links[recordKey]}${recordValue.replace('@', '')}`;
     }
-  }, [isUrlValue, recordKey, recordValue]);
+  }, [isUrlRecord, isUrlValue, recordKey, recordValue]);
 
-  const displayUrl = useMemo(
-    () => (url ? new URL(url).hostname.replace(/^www\./, '') : undefined),
-    [url]
-  );
+  const { displayUrl, displayUrlUsername } = useMemo(() => {
+    const urlObj = url ? new URL(url) : { hostname: '', pathname: '' };
+    return {
+      displayUrl: urlObj?.hostname?.replace(/^www\./, ''),
+      displayUrlUsername: urlObj?.pathname?.replace('/', ''),
```

Comment:
```suggestion
      displayUrlHostname: urlObj?.pathname?.replace('/', ''),
```

Reply:
hm is not the hostname, for example https://twitter.com/estebanmino_ will return estebanmino_

Reply:
oh whoops my bad